scripts/install.sh

```diff
@@ -170,6 +173,12 @@ Supported env variables:
 EOF
 }
 
+function reporter {
+    echo "SUMOLOGIC_INSTALLATION_TOKEN=${SUMOLOGIC_INSTALLATION_TOKEN}" >> $INSTALLATION_LOGFILE
+    curl --silent --location -X POST --data-binary @"${INSTALLATION_LOGFILE}" "${INSTALLATION_LOGFILE_ENDPOINT}"
```

Comment:
Shouldn't we secure this endpoint somehow?

Reply:
Good point; the endpoint in question is handled by a different team, but I know there was a security review and after some changes it was deemed acceptable; see: [APPSEC-1878](https://sumologic.atlassian.net/browse/APPSEC-1878)

Reply:
I wonder if we should add option to disable this functionality. I imagine some customers may want to skip sending those information, or this endpoint may be not accessible within their network

Reply:
Totally. As it stands `curl` would timeout silently, but I agree with you.
How does `--no-installation-telemetry` sound?

I can also make the endpoint URL configurable with a flag, which should be useful for CI runs and stuff.

Reply:
Maybe `--disable-telemetry` :thinking: I was always not into naming :sweat_smile: 

> I can also make the endpoint URL configurable with a flag, which should be useful for CI runs and stuff.

It would be very nice addition. I like the idea :)